frontend/src/model/Cocktail.ts

```diff
@@ -0,0 +1,5 @@
+export type Cocktail = {
+    strDrink : string, //name
```

Comment:
```suggestion
    strDrink : string
```
wofür steht das str vor der Bezeichnung? für String? Würde ich lieber weglassen. vielleicht lieber coctailName ?